controller/tasks.go

```diff
@@ -275,8 +278,23 @@ func (c *Controller) getTaskHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.WriteHeader(http.StatusOK)
-	dagjson.Encoder(task.Representation(), w)
+	if _, set := vars["parsed"]; set {
+		nilStore := func(_ ipld.LinkContext) (io.Writer, ipld.StoreCommitter, error) {
+			b := bytes.Buffer{}
+			return &b, func(_ ipld.Link) error { return nil }, nil
+		}
+		finished, err := task.Finalize(r.Context(), nilStore)
+		if err != nil {
+			log.Errorw("finalize task error", "err", err.Error())
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		w.WriteHeader(http.StatusOK)
+		dagjson.Encoder(finished, w)
+	} else {
+		w.WriteHeader(http.StatusOK)
```

Comment:
I think we don't need this - I seem to recall that the net/http server already replies with a 200 if bytes are written to the response without an explicit response header.